erpcgen/src/templates/c_coders.template

```diff
@@ -63,13 +63,16 @@ if (({$info.sizeTemp} <= {$info.maxSize}) && ({$info.dataTemp} != NULL))
 {% if source == "server" || info.useMallocOnClientSide == true %}
 {$indent}{$info.name} = (uint8_t *) erpc_malloc({$info.maxSize} * sizeof(uint8_t));
 {%  if generateAllocErrorChecks == true %}
-{$indent}if (({$info.name} == NULL) && ({$info.sizeTemp} > 0))
+{$indent}if ({$info.sizeTemp} > 0)
```

Comment:
Still if `generateAllocErrorChecks` is false the issue will happen, suggest to move this check outside the `{% if generateAllocErrorChecks == true %}`